docker-compose.yml

```
@@ -4,25 +4,38 @@ services:
   fhir:
     depends_on:
       - mongo
+      - redis
     build:
       context: .
       dockerfile: Dockerfile
     environment:
+      PUBLIC_BULK_SERVER: http://localhost:3000
       DB_HOST: mongo
       DB_PORT: 27017
       DB_NAME: bulk-export-server
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
```

Suggest moving the default ports to the Dockerfile.
```suggestion
      DB_HOST: mongo
      DB_NAME: bulk-export-server
      REDIS_HOST: redis
```